build/Program.cs

```diff
@@ -0,0 +1,331 @@
+﻿using Microsoft.DotNet.ProjectModel;
+using System;
+using System.IO;
+using System.Linq;
+using NuGet.Versioning;
+using System.Collections.Generic;
+using LibGit2Sharp;
+using Newtonsoft.Json;
+using System.Text;
+
+namespace ConsoleApplication
```

Comment:
StyleCop please.

What's the output of this process? i.e What version numbers are generated?